unit_py/data/linear_two_masses.urdf

```diff
@@ -7,7 +7,13 @@ This is best described by the following diagram:
 -->
 <?xml version="1.0" ?>
 <robot name="linear_masses">
-    <link name="world"/>
+    <link name="world">
+        <inertial>
+            <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
+            <mass value="1.0"/>
```

Comment:
Such a heavy world !

Reply:
I know. Gravity is equal to 0.000000000000001g over there - they often neglect it before Coriolis forces. Eastward deviation is a big problem...